### PR TITLE
update tn_en

### PR DESCRIPTION
**Pull request by rickyw36**

### CODE REVIEW

File: models/microsoft_sdk_en/asr_sdk.py [hunk 1]
```diff
@@ -103,6 +103,11 @@ def recognized(evt):
 
             text = ''
             text = recognize(audio)
+            
+            # delete the space before 's
+            # e.g.  in Apron 's early years  ->  in Apron's early years     
+            text = text.replace(' \'s','\'s')
```
Comment (wukun2015) [inline models/microsoft_sdk_en/asr_sdk.py:109]: Why not leave such text post-processing job to TN tool ?

Comment (dophist) [reply models/microsoft_sdk_en/asr_sdk.py:109]: > 

Wukun, this seems weird to me, it seems only Microsoft reserves an extra space before `'s`.  is this due to the `lexical form` parameter in client code? or it's a universal convention for Azure STT ?

Comment (wukun2015) [reply models/microsoft_sdk_en/asr_sdk.py:109]: i can not reproduce this issue, Jiayu. i don't use en-us in daily work but this issue seems not by design of client or service to me.

Comment (dophist) [reply models/microsoft_sdk_en/asr_sdk.py:109]: > i can not reproduce this issue, Jiayu. i don't use en-us in daily work but this issue seems not by design of client or service to me.

Thanks for the info, Wukun.  Whether A bug or not, we found this processing is necessary for Microsoft to have consistent form of results compared with other APIs.  And it is so ad-hoc and specific that should not be included in general TN module.  I guess we can just keep it here : )

**Pull request merged**